static/app/views/dashboardsV2/widget/metricWidget/index.tsx

```diff
@@ -13,6 +13,7 @@ import {t} from 'sentry/locale';
 import {PageContent} from 'sentry/styles/organization';
 import space from 'sentry/styles/space';
 import {GlobalSelection, Organization, Project} from 'sentry/types';
+import {MetricTag} from 'sentry/types/metrics';
```

Comment:
If we had a re-export this import could be done with the other types 🤷

Reply:
That is a good idea, but perhaps we could do that in a separate PR and maybe also add other types like `types/events` and `types/event`